### PR TITLE
Native teku cleanup

### PR DESCRIPTION
**Pull request by avanheuvelen**
Some things I noticed while setting up an RPL testnode.

### CODE REVIEW

File: src/guides/node/native.md [hunk 1]
```diff
@@ -595,6 +595,11 @@ sudo mkdir /mnt/rpdata/teku_data
 sudo chown eth2:eth2 /mnt/rpdata/teku_data
 ```
 
+Teku needs Java 11 to function, ensure that you have it installed.
```
Comment (jclapis) [inline src/guides/node/native.md:598]: Is this true? The Docker containers use JDK 15 and 16. Do you mean you just need a JDK in general for it?

File: src/guides/node/native.md [hunk 2]
```diff
@@ -675,7 +684,7 @@ Next, we have to give the `rp` user the ability to restart the validator client
 
 Open the `sudoers` file:
 ```
-sudo nano /etc/sudoers
+sudo visudo
```
Comment (jclapis) [inline src/guides/node/native.md:687]: :sweat_smile:

File: src/guides/node/native.md (hunk 1 above)
Comment (avanheuvelen) [reply src/guides/node/native.md:598]: I found this at: https://github.com/ConsenSys/teku#install-prerequisites

Comment (avanheuvelen) [reply src/guides/node/native.md:598]: I have it running on openjdk-11-jre without a problem

**Comment by jclapis** [comment]
Thank you!

**Pull request merged**